Makefile

```diff
@@ -54,4 +57,17 @@ stage:
 
 DIALYZER_APPS = kernel stdlib erts sasl eunit syntax_tools compiler crypto
 
+##
+## deploy to CCM
+##
+deploy:rel
```

Comment:
it installs it to a CCM machine and restarts the service, so `install` or `deploy` make more sense